.env

```diff
@@ -0,0 +1,10 @@
+SCHEME=msauth
```

Comment:
Maybe this should be `AUTH_URL_SCHEME` to match the Auth Connect docs?

Reply:
Done